packages/account-v2/src/modules/src/IDVService/IDVService.tsx

```diff
@@ -71,47 +95,66 @@ export const IDVService = ({ countryCode, handleComplete, onCancel, supportedDoc
 
     return (
         <Formik
-            initialStatus={{ error: '' }}
+            initialStatus={{ error: translateErrorCode(errorStatus, POI_SERVICE.idv) }}
             initialValues={initialValues as TIDVServiceValues}
             onSubmit={handleSubmit}
             validateOnMount
         >
             {({ isSubmitting, isValid, status }) => (
-                <Form className='grid h-full'>
-                    {status?.error && <InlineMessage variant='error'>{status.error}</InlineMessage>}
-                    <div className='grid items-center gap-8 mb-16 grid-cols-[auto_1fr]'>
-                        <Text as='h4' size='md' weight='bold'>
-                            Identity verification
-                        </Text>
-                        <Divider className='block w-full' />
-                    </div>
+                <Form className={twMerge('grid h-full', !countryCode && 'grid-rows-[auto_1fr_auto]')}>
```

Comment:
Do we `twMerge` here? clsx or classnames can do the class joining.

Reply:
I think we are removing clsx since we are using tailwind for styling. So we'll use utility func provided by tailwind and not add additional libs